apps/frontend/src/components/global/upload_tabs/splunk/AuthStep.vue

```diff
@@ -79,18 +83,27 @@ const localHostname = new LocalStorageVal<string>('splunk_hostname');
 export default class AuthStep extends Vue {
   @Prop({type: String, required: false}) indexToShow?: string;
   username = '';
-
   password = '';
   hostname = '';
   index = '';
 
+  // Form required field rule
+  reqRule = (v: string | null | undefined) =>
```

Comment:
this seems like a duplicated function from the aws/auth file.  is it possible to put it in a utility function for use in aws/splunk/tenable files?